parcels/particlesets/soa.py

```diff
@@ -799,7 +799,7 @@ def cstruct(self):
         'cstruct' returns the ctypes mapping of the particle data. This depends on the specific structure in question.
         """
         class CParticles(Structure):
-            _variable_ = [(v.name, POINTER(np.ctypeslib.as_ctypes_type(v.dtype))) for v in self._ptype.variables]
+            _fields_ = [(v.name, POINTER(np.ctypeslib.as_ctypes_type(v.dtype))) for v in self._ptype.variables]
```

Comment:
Changed this back to fields to make the current implementation work. But this is something to look into in the future w.r.t. appropriate naming (as discussed) @CKehl

Reply:
noted #TODO

Reply:
ah, if you're extra-awesome, you could add a 
```
#TODO adapt naming of _fields_ in CParticles (code generator) into _variables_
```
at that code-step, which would then highlight at least in PyCharm